hw1.py

```diff
@@ -0,0 +1,62 @@
+##Complementator##
+compl_dna = {'A': 'T', 'C': 'G', 'G': 'C', 'T': 'A', 'a': 't', 'c': 'g', 'g': 'c', 't': 'a'}
+compl_rna = {'A': 'U', 'C': 'G', 'G': 'C', 'U': 'A', 'a': 'u', 'c': 'g', 'g': 'c', 'u': 'a'}
+transcr = {'A': 'U', 'C': 'G', 'G': 'C', 'T': 'A', 'a': 'u', 'c': 'g', 'g': 'c', 't': 'a'}
+# create DNA or RNA complement sequence. If not stated explicitly, creates DNA
+def complement(seq):
+    for se in seq:
+        if (se not in compl_dna) &  (se not in compl_rna):
+            # print("Invalid symbol", letter,". Try again!")
+            return "Invalid symbol(s)! Try again!"
+        # else:
+        #   continue
+    if 'u' in seq.lower() and 't' in seq.lower():
```

Comment:
Если букв немного, то так норм, а вообще для этого очень удобно использовать множества
```suggestion
    if {'u', 't'} <= set(seq.lower()):
```
<= - оператор проверки на подмножество